### DISCUSSION TITLE
Support for Skopeo signing/verification

### DISCUSSION THREAD

**Discussion by zosocanuck** [sse-secure-systems/connaisseur#385] [Ideas]
Hi,

Are there any plans to support skopeo signature verification?

**Comment by xopham** [comment]
Hi @zosocanuck , that would be based on [RedHat Container Signing](https://www.redhat.com/en/blog/container-image-signing), right? While I was aware of it, I am not certain how much traction it has. Is there a considerable user base? Do you have some more information on your use case?
It also seems very similar to the [Cosign/Sigstore](https://github.com/sigstore/cosign). What is the advantage of choosing it over cosign?

In general, Connaisseur works as a container image validator and was specifically built to be compatible with multiple solutions to allow validation of images from different sources and signature solutions in the cluster. I would consider that an interesting idea and the signing schemes appears rather simple and handy.
